packages/maskbook/src/plugins/Ethereum/UI/GasNowDialog.tsx

```diff
@@ -0,0 +1,229 @@
+                            )}
+                            key={i}>
+                            <div className={classes.head}>
+                                <Typography className={classes.title}>{option.title}</Typography>
+                                <FiberManualRecordIcon
+                                    className={classNames(classes.dot, i === select ? classes.dotSelector : undefined)}
+                                />
+                            </div>
+                            {gasNow || i === 2 ? (
+                                <>
+                                    <div>
+                                        {i == 2 ? (
+                                            <div className={classes.gweiBox}>
+                                                <TextField
+                                                    className={classes.customInput}
+                                                    value={customGas}
+                                                    onChange={(e) => {
+                                                        const v = e.target.value
+                                                        setCustomGas(isNaN(Number(v)) ? '' : v)
+                                                    }}
+                                                />
+                                                <Typography className={classes.gwei}>
+                                                    {t('plugin_gas_now_dialog_gwei')}
+                                                </Typography>
+                                            </div>
+                                        ) : (
+                                            <Typography className={classes.gwei}>
+                                                {t('plugin_gas_now_dialog_gwei', {
+                                                    gasPrice: formatWeiToGwei(option.gasPrice),
+                                                })}
+                                            </Typography>
+                                        )}
+                                        {usdRate ? (
+                                            <Typography className={classes.estimate}>
+                                                {t('plugin_gas_now_dialog_usd', {
+                                                    usd: new BigNumber(option.gasPrice)
+                                                        .div(10 ** 18)
+                                                        .times(usdRate)
+                                                        .toPrecision(3),
+                                                })}
+                                            </Typography>
+                                        ) : null}
+                                    </div>
+                                    <Typography>{i == 2 ? customEstimateTime : option.time}</Typography>
```

Comment:
Don't forget to use color. Otherwise, it will render text the wrong color in the dark mode.